lib/launcher.rb

```diff
@@ -102,13 +102,19 @@ def https_app(https_port, plugins = https_plugins)
         :SSLVerifyClient => OpenSSL::SSL::VERIFY_PEER,
         :SSLPrivateKey => load_ssl_private_key(settings.ssl_private_key),
         :SSLCertificate => load_ssl_certificate(settings.ssl_certificate),
-        :SSLCACertificateFile => settings.ssl_ca_file,
+        :SSLCACertificateFile => verify_ssl_ca_file(settings.ssl_ca_file),
```

Comment:
Would it make sense to do it earlier? Like after line 67?

Reply:
Yeah I tend to prefer to do just a single if statement in this method, unless there are plans to add additional verification.